docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -1,43 +1,23 @@
 [id="using-new-subnet-ranges_{context}"]
 
-= Scenario 1: Using new subnet ranges
+= Configuring new subnet ranges
 
-This scenario is compatible with any existing subnet configuration, and can be
-used even when the existing cluster subnet ranges don't have enough free IP
-addresses for the new control plane services.
+You can define new IP ranges for control plane services that belong to a different subnet that is not used in the existing cluster. Then you configure link local IP routing between the old and new subnets to enable old and new service deployments to communicate. This involves using the {OpenStackPreviousInstaller} mechanism on a pre-adopted cluster to configure additional link local routes. This enables the EDP deployment to reach out to adopted nodes by using the old subnet addresses. You can use new subnet ranges with any existing subnet configuration, and when the existing cluster subnet ranges do not have enough free IP addresses for the new control plane services.
```

Comment:
EDP = EDPM? (We use “data plane” instead of EDPM).

Reply:
Yes, `data plane`.

---

The "old" term is probably not the best because they are still used in the "adopted" / "new" environment. They are subnets that were used before adoption, but they are still current / used in the new.

Reply:
"Existing" subnets might be better.